pyproject.toml

```diff
@@ -24,12 +24,12 @@ dependencies = [
   "h5py>3.12.1",
   "libsupermesh",
   "loopy>2024.1",
-  "petsc4py==3.23.3",
+  "petsc4py==3.23.4",
```

Comment:
I think something has gone wrong with the merge here. These should be commented out on `master`.